stat_unix.go

```diff
@@ -0,0 +1,117 @@
+// +build !plan9
```

Comment:
I would expect a `_unix.go` file would also exclude `!windows`, and likely `!js` as well. But then they are compatible with this code only because Go provides “fictitious” values so that they are otherwise compable without code changes like we‘ve had to do here for plan9…

Not sure I have an answer for what to do here, though. I think technically, it‘s `_posix.go`. But only because it‘s the part of POSIX that we‘re able to build support for out of the very rudimentary baseline WIndows POSIX support… Maybe `_posix_compat.go`? or just `_compat.go`?

Reply:
I'll use `_posix.go` because that's what the standard library uses.

Reply:
Done